src/map/handler/Map.Keyboard.js

```diff
@@ -141,8 +141,8 @@ export var Keyboard = Handler.extend({
 		if (e.altKey || e.ctrlKey || e.metaKey) { return; }
 
 		var key = e.keyCode,
-		    map = this._map,
-		    offset;
+		map = this._map,
+		offset;
```

Comment:
Minor nitpick: Could you revert the white space here? It was 2 tabs of indentation plus 4 spaces of alignment. Thanks!

Reply:
Spaces re-added.
For some reason when I first ran `jake test` I got 2 `Expected indentation of 2 tab characters but found 4` errors on line 144 & 145. Also linting errors in my text editor, therefore I just fixed the spacing as per the linter. 
Added back the 4 additional spaces of indentation and tests still passing.